lib/bitcoin/cw_bitcoin.dart

```diff
@@ -287,6 +287,10 @@ class CWBitcoin extends Bitcoin {
   }) async {
     List<DerivationInfo> list = [];
 
+    if (!validateMnemonic(mnemonic) && !bip39.validateMnemonic(mnemonic)) {
+      throw BitcoinMnemonicIsIncorrectException();
+    }
```

Comment:
did you try reproducing it again after fixing it?
cuz there is no try/catch block all the way to the UI that will catch this exception

Reply:
yeah the fix definitely works

the main reason it's implemented this way is to match the behavior of the other wallet types but I'm also trying to minimize code duplication here
I think most of the time you're right doing form validation in the UI layer makes sense, but since we already validate (or should be if we aren't) mnemonics / seeds in the restoreFromSeed() / restoreFromKeys() functions in each wallet type, I don't see an issue with having that error be passed through to the ui

Reply:
nope, my point is, if you try to reproduce this issue, you will get the `oops we got an error` popup and send a mail to support, while we don't need that, we just need to inform the user that the seeds are incorrect

Reply:
oooh that makes sense